tests/foreman/longrun/test_provisioning_computeresource.py

```diff
@@ -255,12 +266,11 @@ def test_positive_provision_vmware_with_host_group(self):
                 'provision-method': 'build',
             }
         )
-        hostname = '{}.{}'.format(host_name, self.config_env['domain'])
-        self.assertEquals(hostname, host['name'])
+        hostname = '{}.{}'.format(host_name, provisioning['config_env']['domain'])
```

Comment:
f' strings ?

Reply:
I just rewrote to pytest, perhaps I'll mass-port to f-strings later.

Reply:
Hi, take a look at https://github.com/SatelliteQE/robottelo/pull/8281